crates/autopilot/src/solvable_orders.rs

```diff
@@ -590,12 +590,16 @@ async fn filter_unsupported_tokens(
     Ok(orders)
 }
 
-fn filter_limit_orders_with_insufficient_sell_amount(mut orders: Vec<Order>) -> Vec<Order> {
-    // Unwrap because solvable orders always have a surplus fee.
-    orders.retain(|order| match &order.metadata.class {
-        OrderClass::Limit(limit) => order.data.sell_amount > limit.surplus_fee.unwrap(),
-        _ => true,
-    });
+fn filter_fok_limit_orders_with_insufficient_sell_amount(mut orders: Vec<Order>) -> Vec<Order> {
+    // Unwrap because solvable fok orders always have a surplus fee.
+    orders.retain(
+        |order| match (&order.metadata.class, order.data.partially_fillable) {
+            (OrderClass::Limit(limit), false) => {
+                order.data.sell_amount > limit.surplus_fee.unwrap()
```

Comment:
```suggestion
                order.data.sell_amount >= limit.surplus_fee.unwrap()
```

Reply:
I think it should be `>`, otherwise we have 0-sell amount orders which we don't want.

Reply:
This PR doesn't change this condition.

Reply:
> This PR doesn't change this condition btw.

Yes. Just wanted to point out a possible mistake that was maybe overlooked earlier.

> I think it should be >, otherwise we have 0-sell amount orders which we don't want.

I mean this is difference is negligible so I think both options are okay but since solvers could theoretically settle the orders I thought it made sense to include them.
I'm fine with leaving it as is.